docs/tasks/debug-application-cluster/core-metrics-pipeline.md

```diff
@@ -0,0 +1,41 @@
+---
+approvers:
+- fgrzadkowski
+- piosz
+title: Core metrics pipeline
+---
+
+Starting from Kubernetes 1.8, resource usage metrics, such as container CPU and memory usage,
+are available in Kubernetes though the the Metrics API. These metrics can be either accessed directly
+by user, for example by using `kubectl top` command, or used by a controller in the cluster, e.g.
+Horizontal Pod Autoscaler to make decisions.
+
+## The Metrics API
+
+Through the Metrics API you can get the amount of resource currently used
+by a given node or a given pod. This API doesn't store the metric values,
+so it's not possible for example to get the amount of resources used by a
+given node 10 minutes ago.
+
+The API no different from any other API:
+it is discoverable through the same endpoint as the other Kubernetes APIs under `/apis/metrics.k8s.io/` path.
+It also offers the same security, scalability and reliability guarantees.
+
+The API is defined in [k8s.io/metrics](https://github.com/kubernetes/metrics/blob/master/pkg/apis/metrics/v1beta1/types.go)
+repository. You can find more information about the API resources there.
+
+## Metrics Server
+
+[Metrics Server](https://github.com/kubernetes-incubator/metrics-server) is a cluster-wide aggregator of resource usage data.
+Starting from Kubernetes 1.8 it's deployed by default in clusters crated by `kube-up.sh` script
+as a Deployment object.
+User can deploy it also using the provided [deployment yamls](https://github.com/kubernetes-incubator/metrics-server/tree/master/deploy).
```

Comment:
AFAIR the recommendation is to talk to user instead of referencing it in a third person, i.e. `You can deploy...`

Please remove `also`

Why new line? That'll end up as one paragraph anyway

Reply:
done